### DISCUSSION TITLE
关于使用pjax音乐界面后的bug

### DISCUSSION THREAD

**Discussion by Moriic** [YunYouJun/hexo-theme-starry#143] [Q&A 疑难杂症] [answered]
**我测试过多次使用pjax 和不使用pjax 后     确实是使用pjax 后会出现以下问题**

**1.第一次打开含播放列表的网页会显示不出播放器，刷新之后可显现播放器**

![D9DL24Q54A6_@LHA$W4} {0](https://user-images.githubusercontent.com/62750805/127309836-0d3eeddf-514f-43f9-9a52-02311952f107.png)
![YGW6W(EUX@{6TKZ6 Z`37Q8](https://user-images.githubusercontent.com/62750805/127309839-a9536529-3f71-4295-abb1-d1f23fa54c0e.png)

**2.进入含播放列表的网页后回到主页会出现字体不显示**
![%4_KWTYU`O6 A_V1ETECRN](https://user-images.githubusercontent.com/62750805/127309921-57f63f58-2293-47f5-a79f-f6ae2fe375a2.png)

**3.进入含播放列表的页面后点侧边栏 点进头像页面 后侧边栏不会自动缩回**
![LOGB)C@P{2C~ `_SURW2C4T](https://user-images.githubusercontent.com/62750805/127310136-069d1c5f-4e52-4696-81b6-76bc75a19450.png)
**

**Comment by Moriic** [comment]
这是我的网站 www.cwcn.tk

**Comment by YunYouJun** [comment, marked as answer]
问题 1 的播放器 pjax 问题已修复，问题 2 此前也已修复，请更新至最新版 v1.6.2，问题 3 是 pjax 的特性不是 BUG。

**Reply by Moriic** [reply]
感谢😊

**Reply by Moriic** [reply]
**。。。我已更新到1.6.2
但是问题还是存在是什么鬼。。。。😂😂**

**Reply by YunYouJun** [reply]
> **。。。我已更新到1.6.2 但是问题还是存在是什么鬼。。。。😂😂**

请提供更详细的描述，和可复现仓库。

**Reply by Moriic** [reply]
en  就是我更新完后还是出现了我上述说的关于pjax中的问题1和2，这是我的博客www.cwcn.tk
上传到github的仓库：https://github.com/chen2020I/chen2021.github.io

**Reply by YunYouJun** [reply]
> en 就是我更新完后还是出现了我上述说的关于pjax中的问题1和2，这是我的博客www.cwcn.tk
> 上传到github的仓库：https://github.com/chen2020I/chen2021.github.io

这不是没问题吗……

**Reply by Moriic** [reply]
打扰了是我位置放错了🤣
感谢感谢

**Reply by Mirukuuun** [reply]
> > **。。。我已更新到1.6.2 但是问题还是存在是什么鬼。。。。😂😂**
> 
> 请提供更详细的描述，和可复现仓库。

您好，我1.6.2也遇到了同样的问题，不知道时哪里出现了问题QAQ
还有  代码的复制按钮，也需要刷新之后才会出现，
首页的“一言”在第二次打开网站后就不出现了，需要在sidebar里面点击首页按钮才会再正常出现
可以麻烦有空的时候帮我看一下嘛🙏
我的网址：https://mirukuuun.github.io/
github的仓库：https://github.com/Mirukuuun/Mirukuuun.github.io

**Reply by YunYouJun** [reply]
问题还存在吗，有的话用 Issue 反馈吧。

**Reply by showmaker-hub** [reply]
> 问题还存在吗，有的话用 Issue 反馈吧。

您好，更新1.70后，问题2和3已经解决，但是还差问题1和附加问题也就是(url)https://github.com/YunYouJun/hexo-theme-yun/issues/195

博客地址：https://showmaker-hub.vercel.app/
仓库地址：https://github.com/showmaker-hub/hexo-blog

主题1.70后，复现过程
1.打开后侧边栏里有个网页云音乐的图标，这个是抄你的音乐界面，我想把这个问题解决了再修改。刷新后才能显示播放器
2.打开相册，点击私密相册，刷新后输入密码才有反应，否则点击验证密码无反应（密码test）
